docs/DafnyRef/Attributes.md

```diff
@@ -33,7 +33,7 @@ with no argument is interpreted as if it were true.
 
 ## 23.1. Attributes on top-level declarations
 
-### 23.1.1. `{:autocontracts}` {#sec-attributes-autocontracts}
```

Comment:
Why remove this one?

Reply:
Huh -- must have been an accidental artifact of merging/conflict resolution. I will leave it in.